dask/verify_dask_yarn.py

```diff
@@ -1,13 +1,6 @@
-from dask_yarn import YarnCluster
-from dask.distributed import Client
 import dask.array as da
-import dask_ml
-import dask_bigquery
 
 import numpy as np
 
-cluster = YarnCluster()
```

Comment:
This is the crux of the test.  We can't remove it and still call this a test.

Reply:
This variable is not being used in the script. It is only declared.

Reply:
That's not a declaration, that's an instantiation